modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -98,8 +98,15 @@ IMPORTANT: You cannot change the service network address block during the migrat
 * Geneve (Generic Network Virtualization Encapsulation) overlay network port
 --
 +
+[IMPORTANT]
+====
+You must configure hybrid networking with OVN-Kubernetes during the installation of your cluster. You cannot switch to hybrid networking after the installation process.
+
+In addition, the hybrid  OVNKubernetes cluster is a requirement for Windows Machine Config Operator (WMCO)
```

Comment:
Should this also be "OVN-Kubernetes cluster" for consistency?

Also, re "for Windows Machine Config Operator (WMCO)", I'd say "for the Windows Machine Config Operator (WMCO)." (don't forget the period at the end of the sentence 😉 )

For Operators, I'd recommend adding an attribute for the name to save you from typing it out each time it's re-used, but totally up to you if you want to do that.

Reply:
@JoeAldinger 

We're going to rebrand this for 4.12+, but in the meantime, I'd say

> You must configure hybrid networking with the OVN-Kubernetes cluster network provider...

> In addition, an {product-title} cluster configured with the OVN-Kubernetes cluster network provider is a...

I think.